doc/src/quick_start_guide.md

```diff
@@ -24,11 +24,13 @@ regex = "1"
 # Add a build-time dependency on the lalrpop library:
 [build-dependencies]
 lalrpop = "0.19.0"
-# If you do not supply your own, external lexer you also need the `lexer` feature
-# lalrpop = { version = "0.18.1", features = ["lexer"] }
+# If you are supplying your own external lexer you must disable default features so that the
+# built-in lexer feature is not included
```

Comment:
```suggestion
# If you are supplying your own external lexer you can disable default features so that the
# built-in lexer feature is not included
```